docs/tooling.md

```diff
@@ -25,17 +25,19 @@ In particular:
 
 - `leafwing-input-manager` and `bevy_mod_picking` are very likely to be upstreamed into Bevy in the near future.
 - `bevy-inspector-egui` and `bevy_mod_debugdump` help fill the gap until Bevy has its own editor.
-- `bevy_rapier` or `avian` helps fill the gap until Bevy has its own physics engine.
+- `avian` or `bevy_rapier` helps fill the gap until Bevy has its own physics engine. `avian` is easier to use, `bevy_rapier` has better performance.
 - `sickle_ui` is well-aligned with `bevy_ui` and helps fill the gap until Bevy has a full collection of UI widgets.
 
+None of these are necessary, but they can save you a lot of time and effort.
+
 ## VS Code extensions
 
 If you're using [VS Code](https://code.visualstudio.com/), the following extensions are highly recommended:
 
-| Name                                                                                                      | Description             |
-|-----------------------------------------------------------------------------------------------------------|-------------------------|
-| [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer)              | Rust support            |
-| [Even Better TOML](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml)          | TOML support            |
-| [vscode-ron](https://marketplace.visualstudio.com/items?itemName=a5huynh.vscode-ron)                      | RON support             |
-| [crates](https://marketplace.visualstudio.com/items?itemName=serayuzgur.crates)                           | `Cargo.toml` support    |
-| [EditorConfig for VS Code](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) | `.editorconfig` support |
+| Name                                                                                                      | Description                       |
+|-----------------------------------------------------------------------------------------------------------|-----------------------------------|
+| [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer)              | Rust support                      |
+| [Even Better TOML](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml)          | TOML support                      |
+| [vscode-ron](https://marketplace.visualstudio.com/items?itemName=a5huynh.vscode-ron)                      | RON support                       |
+| [Dependi](https://marketplace.visualstudio.com/items?itemName=fill-labs.dependi)                          | `crates.io` dependency resolution |
```

Comment:
I couldn't find this extension within VS Code (maybe I have to restart?), but I do see the migration notice on the `crates` repo.

Reply:
![image](https://github.com/user-attachments/assets/91a48db0-d29f-4526-827f-8b69d34fda63)

Hmm, dunno

Reply:
Won't block on this, I'll try and figure this out.